lib/animina_web/live/root_live.ex

```diff
@@ -111,7 +111,8 @@ defmodule AniminaWeb.RootLive do
               type: :text,
               required: true,
               autofocus: true,
-              autocomplete: :username
+              autocomplete: :username,
+              "phx-debounce": "300"
```

Comment:
Just for curiosity: What is `debounce`?

Reply:
Debounce adds a delay for any events that get fired as the user is typing. For example validations will not happen immediately.